### DISCUSSION TITLE
How to generate package.json in dist folder when packaging

### DISCUSSION THREAD

**Discussion by ngoisaomouoc555** [sveltejs/kit#11717] [Q&A]
![1-5](https://github.com/sveltejs/kit/assets/111834488/47cd4f4b-c8e9-48f2-9945-64016f2bd487)

![image](https://github.com/sveltejs/kit/assets/111834488/15b2475a-cff9-495b-8b10-d20d09c8bd0c)

Hello,
I'm trying to publish the project like package so I used svelte-package command.
I'm getting the error by package.json file and it's not generated in `dist` folder.
I attached the error text and my package.json.
Please let me know what I missed.
Thank you.

**Comment by teemingc** [comment]
I believe you can simply run `npm publish` without targeting the `dist` folder. This will include your package.json and all files in the `dist` folder as specified by the `files` key in your `package.json`.
